recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "apache-beam" %}
-{% set version = "2.55.1" %}
+{% set version = "2.56.0" %}
 
 package:
   name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b47e2c9a10882557a83da07f1ba118cbdab869d09189057ef4a006354a3e110b
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 4d9f38ff287757a3d7045a96ecad4bf825e2fb1ca0b9da334927d7cc8b75b250
 
 build:
-  skip: true  # [win or py>=312]
+  skip: true  # [win or py>=313]
```

Comment:
While there is a warning that 3.12 isn't fully tested, it also isn't explicitly excluded:
https://github.com/apache/beam/blob/master/sdks/python/setup.py#L288-L292

Reply:
IIUC there are some gaps related 3.12 that is being fixed for the next Beam version.